### ISSUE TITLE
New employers landing page

### ISSUE THREAD

**Issue by rubenkostucki** [makersacademy/website#516]
As an employer
So that I can be more easily persuaded to hire grads
I want a new landing page that explains to me the benefits of hiring from Makersx

On branch ruben-landingpages

Couple of blockers:

1. I'd like that instead of 'City' on the page, its picks up the location based on the browser
2. I believe there's a slight misalignment of a CTA on mobile, unlinked to the work I did
3. Blocked by #511

**Comment by rubenkostucki** [comment]
I was thinking of building numerous landing pages for say Manchester, Berlin, Paris etc. The problem is we will have a lot of duplicate content which is not good for SEO.

I'd like that we change the "City's" for the best guess of the geolocation of the person browsing. If it's a small town or we can't find anything else than UK, then that's fine too. Happy to discuss this.

Why is this important? Hiring is a localised action/behaviour.

**Comment by rubenkostucki** [comment]
When narrowing my chrome browser to check how it looks on mobile, it seems that the 'start hiring' CTA is not rightly centred. I added two more logos than the usual partial. Yet, when I remove those and use the partial as I copied it from other places, it doesn't seem to affect the location of the CTA.

<img width="400" alt="screen shot 2016-02-08 at 11 35 22" src="https://cloud.githubusercontent.com/assets/2648116/12884176/ce7bb066-ce5c-11e5-8319-2113ee7a21f0.png">

**Comment by pitchinvasion** [comment]
@rubenkostucki how crucial is the geolocation to the landing page? That's going to be quite hard to get right, I'd argue that's a seperate ticket if we can launch a generic landing page we can add the custom content later

**Comment by rubenkostucki** [comment]
Hiring, I believe, is a localised event. If I need to hire in Manchester, I want to know MA have people in Manchester. But OK, I'll remove any kind of reference to location, push the new code and create a new ticket @pitchinvasion 

Point 2 and 3 are still relevant though. I can't go live without the bug being checked and ideally not without the typeform integration.

**Comment by rubenkostucki** [comment]
Will has had time to review the content so we're all ready here from a Careers POV. It's now a totally generic page, with no mention of cities.

Will and I discussed actually making this LP the main LP for /employers and split testing it with the current one (Will's version). I'd like to discuss this with the engineers who will help us complete this task.

**Comment by spike01** [comment]
Redirect split test on Optimity -> split traffic to landing pages?

**Comment by pitchinvasion** [comment]
Yes. But this is now blocked by #511 and #514

**Comment by rubenkostucki** [comment]
All good to complete this today @pitchinvasion @danldb ? :)
